src/components/ha-sidebar.ts

```diff
@@ -474,15 +474,14 @@ class HaSidebar extends LitElement {
 
       .menu {
         box-sizing: border-box;
-        height: 65px;
+        height: 64px;
```

Comment:
Do we still need this now we don't have a border on the other toolbar?

Reply:
And if we do, the height doesn't match the config headers now?

Reply:
Yep you're right... changed it back to 65px

Reply:
But just checking... and the reason i started restyling .menu and .app-header was becease as it is now... (as it was) .menu is 65px height and app-toolbar .64px.. so theres a difference... doesnt look very nice. What do you think?
![VirtualBox_HOMEASSISTANT_08_04_2020_16_38_41](https://user-images.githubusercontent.com/11946839/78797317-d358a700-79b7-11ea-8a8e-a3e7dd547650.png)

Reply:
The pixel difference is the border on the sidebar menu, give that the color of the background of the sidebar and you should be fine.

Reply:
I'm ok with it being 64px, as the border is transparent by default, but you should change the other pages so they match.

Reply:
(and it is used in a lot of places, also these: 
![image](https://user-images.githubusercontent.com/5662298/78798274-16674a00-79b9-11ea-95d5-dcb4ed22756d.png))

Reply:
If you are going to change it all anyway, make it a variable that we use to calculate the height of the content and the header.

Reply:
oohh thats not cool haha... uhmmm thinking

Reply:
Made some changes;
Added  sidebar-header-height: "64px" and app-header-height: "64px" and implemented it. 
Looks like it's good... Also looks good on the config pages.
Let me know if i forgot something? Thanks!